README.md

```diff
@@ -132,3 +133,18 @@ A pre-built windows installer will be available soon. Until then it isn't too ha
       * The `-InstallFFMPEG` flag will automatically pull the stable ffmpeg binaries appropriate to your architecture (x86/x64 only for now) from [Zeranoe](https://ffmpeg.zeranoe.com/builds/) and place them in your Jellyfin directory. 
 0. (Optional) Use [NSSM](https://nssm.cc/) to configure JellyFin to run as a service
 0. Jellyfin is now available in the default directory (or the directory you chose). Assuming you kept the default directory, to start it from a Powershell window, run, `&"$env:APPDATA\Jellyfin-Server\EmbyServer.exe"`. To start it from CMD, run, `%APPDATA%\Jellyfin-Server\EmbyServer.exe`
+
+### Docker (also for Raspberry Pi)
+
+0. Clone Jellyfin into a directory of your choice.
+    ```
+    git clone https://github.com/jellyfin/jellyfin.git C:\Jellyfin
```

Comment:
Are you sure this path should be "C:\Jellyfin"? :)

Reply:
I don't think so. Got in a hurry sorry